src/LandIce/evaluators/LandIce_BasalFrictionCoefficient.hpp

```diff
@@ -55,14 +55,15 @@ class BasalFrictionCoefficient : public PHX::EvaluatorWithBaseImpl<Traits>,
   ScalarT printedLambda;
   ScalarT printedQ;
 
-  double given_val;  // Constant value (for CONSTANT only)
+  double beta_val;   // beta value [kPa yr/m] (for CONSTANT only)
+  MeshScalarT N_val; // effective pressure value [kPa]
 
   // Input:
   PHX::MDField<const RealType>          BF;
   PHX::MDField<const VelocityST>        u_norm;          // [m yr^{-1}]
   PHX::MDField<const ParamScalarT>      lambdaField;     // [km], characteristic length
   PHX::MDField<const ParamScalarT>      muField;         // [yr^q m^{-q}] or [adim], Power Law with exponent q, Coulomb Friction
-  PHX::MDField<const EffPressureST>     N;               // [kPa]
+  PHX::MDField<const MeshScalarT>       N;               // [kPa]
```

Comment:
For hydrology-coupled simulations, you do need N to be ScalarT. That's why I had the EffPressureST template arg. Does this code compile for you?

Reply:
Oh.. right.. yeah, let me fix this. It does compile though

Reply:
Ah, I wonder how/why. This line in LandIce_Hydrology.hpp should cause an error:

```
ev = Teuchos::rcp(new BasalFrictionCoefficient<EvalT,PHAL::AlbanyTraits,ScalarT,RealType,RealType>(*p,dl));
```

Reply:
I think I had changed that line. Anyway, I reverted the change